src/User.php

```diff
@@ -5738,7 +5738,7 @@ public function showForgetPassword($email)
             Session::addMessageAfterRedirect($e->getMessage(), false, ERROR);
             return;
         }
-        Session::addMessageAfteRredirect(__('If the given email address match an exisiting GLPI user, you will receive an email containing the informations required to reset your password. Please contact your administrator if you do not receive any email.'));
+        Session::addMessageAfteRredirect(__('If the given email address corresponds to one and only one GLPI user, you will receive an email containing the information required to reset your password. Please contact your administrator if you do not receive emails.'));
```

Comment:
```suggestion
                __('If the given email address corresponds to one and only one GLPI user, you will receive an email containing the information required to reset your password. Please contact your administrator if you do not receive an email.')
```

Reply:
I guess there's a bad copy/paste?:
```suggestion
        Session::addMessageAfteRredirect(__('If the given email address corresponds to one and only one GLPI user, you will receive an email containing the information required to reset your password. Please contact your administrator if you do not receive an email.'));
```

Reply:
Yeah 🙁. I've had to work on mobile and from a laptop the last few days so it's not been the best.

The first suggestion I made should apply to both cases